website_form_recaptcha/models/website_config_settings.py

```diff
@@ -1,15 +1,34 @@
 # Copyright 2019 Simone Orsi - Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     recaptcha_key_site = fields.Char(
         related='website_id.recaptcha_key_site',
+        readonly=False,
     )
     recaptcha_key_secret = fields.Char(
         related='website_id.recaptcha_key_secret',
+        readonly=False,
+    )
+
+
+    @api.depends('website_id')
+    def has_google_recaptcha(self):
```

Comment:
This method should be private (start by underscore), and never be called with the same name as a field. Call it `_compute_has_google_recaptcha`.